### ISSUE TITLE
Unable to login using webview, login form doesnt appear

### ISSUE THREAD

**Issue by AMajor-C** [nextcloud/talk-android#3386]
### Steps to reproduce

1. install app
2. try to login
3. ???

### Expected behaviour

Possibility to login

### Actual behaviour

If click to login button nothing happens. If click on link to nextcloud root page login form doesnt appear only backgroud, logo and footer visible.

### Device brand and model

Sharp SH-02L

### Android version

8.1

### Nextcloud Talk app version

17.20.0 alpha 3

### Nextcloud server version

27.1.2

### Talk version

17.1.1

### Custom Signaling server configured

None

### Custom TURN server configured

None

### Custom STUN server configured

None

### Additional information

Seems even if webview last installed your app uses old outdated that installed in system.
May be its time to drop dependence on webview to trashcan ? Ah easier to drop support another android version than do some good auth method. Really easier and/or fast doesnt mean "best" or even good.

**Comment by mahibi** [comment]
@AMajor-C 

what is the version of your Android System WebView? How did you install/updated it?

**Comment by AMajor-C** [comment]
> @AMajor-C
> 
> what is the version of your Android System WebView? How did you install/updated it?

Updated via gplay to 117.0.5938.153
System default: 67.0.3396.127

**Comment by AMajor-C** [comment]
btw i had same problem on android 5.1 device

**Comment by hello-world-dot-c** [comment]
Same problem here, Android 9, WebView is at 125.0.6422.147 and Chrome is latest. No difference if I change the system default web browser to Chrome or Firefox. No error on the server and all other clients work fine. On this device, I have already Nextcloud and Notes clients running no problem and on other devices I have the Android Talk client running no problem, using the same server.

**Comment by TechnologyMan101** [comment]
I have this issue as well. This is on an Android 10 device with WebView 74.0.3729.186 which unfortunately cannot be updated. One thing I should mention is that the Nextcloud Files app is able to log in, most likely because it opens my default browser (Firefox) to go through the login process.

**Comment by andynameistaken** [comment]
Same issue: Android 9

**Comment by mahibi** [comment]
> Nextcloud Files app is able to log in, most likely because it opens my default browser (Firefox) to go through the login process.

yes that is planned for the talk app as well and will avoid the bug. However the timeline for this is not yet set
